pennylane/ops/op_math/linear_combination.py

```diff
@@ -45,6 +45,8 @@ class LinearCombination(Sum):
             Can be ``'qwc'`` (qubit-wise commuting), ``'commuting'``, or ``'anticommuting'``.
         method (str): The graph coloring heuristic to use in solving minimum clique cover for grouping, which
             can be ``'lf'`` (Largest First) or ``'rlf'`` (Recursive Largest First). Ignored if ``grouping_type=None``.
+        grouping_indices (Optional[List[List[int]]]): Which terms can be computed at the same time. Sets
+            the corresponding property. Takes precedence over ``grouping_type`` and ``method``.
```

Comment:
> Takes precedence over ``grouping_type`` and ``method``.

Just to have mentioned this: This allows the op to have a `grouping_type`/`method` setup that is incompatible with the set `grouping_indices`, right? Could this cause any trouble? Do we want to catch the case where users pass non-trivial `grouping_type` and `grouping_indices` at the same time?

Reply:
We could potentially error if both `grouping_type` and `grouping_indices` are specified?